Anima/diffusion/mcm_estimator/animaMCMEstimatorImageFilter.hxx

```diff
@@ -1150,16 +1123,13 @@ MCMEstimatorImageFilter<InputPixelType, OutputPixelType>
             else if (m_Optimizer == "bfgs")
                 tmpOpt->SetLocalOptimizer(NLOPT_LD_LBFGS);
 
-            if (!m_UseFixedWeights)
-            {
-                typedef anima::MCMWeightsInequalityConstraintFunction WeightInequalityFunctionType;
-                WeightInequalityFunctionType::Pointer weightsInequality = WeightInequalityFunctionType::New();
-                double wIneqTol = std::min(1.0e-16, xTol / 10.0);
-                weightsInequality->SetTolerance(wIneqTol);
-                weightsInequality->SetMCMStructure(costCast->GetMCMStructure());
+            typedef anima::MCMWeightsInequalityConstraintFunction WeightInequalityFunctionType;
+            WeightInequalityFunctionType::Pointer weightsInequality = WeightInequalityFunctionType::New();
+            double wIneqTol = std::min(1.0e-16, xTol / 10.0);
```

Comment:
What a nice formula! Wouldn't it be cleaner to use something like `sqrt(std::limits<double>::epsilon())` instead of hard `10^(-16)` ?

Reply:
We could indeed. What's the value of std::limits<double>::epsilon() ? (and why sqrt of it ?)